README.md

```diff
@@ -1,6 +1,6 @@
 # Task Service Deep Learning Tools
 
-At LifeOmic, we make it easy to train deep learning models on our [task service](https://docs.us.lifeomic.com/user-guides/task-service/).
+At LifeOmic, we make it easy to train deep learning models on our [task service](https://docs.dev.lifeomic.com/user-guides/automation/tasks/).
```

Comment:
Pointing at the `dev` documentation doesn't seem right. Can you use `https://docs.us.lifeomic.com/user-guides/automation/tasks/` instead?